tests/integration_tests/test_assets/pydantic/compiled_prompt_reask_2.txt

```diff
@@ -21,11 +20,9 @@ Help me correct the incorrect values based on the given error messages.
 Given below is XML that describes the information to extract from this document and the tags to extract it into.
 
 <output>
-    <list name="people" description="A list of 3 people.">
-        <object description="Information about a person." pydantic="Person">
-            <string name="name" description="The name of the person."/>
-            <integer name="age" description="The age of the person." format="age-must-be-between-0-and-150"/>
-            <string name="zip_code" description="The zip code of the person." format="zip-code-must-be-numeric; zip-code-in-california"/>
+    <list name="people">
+        <object>
+            <string name="zip_code" format="zip_code_must_be_numeric; zip_code_in_california"/>
```

Comment:
These removals reflect the changes from a partial reask instead of a full reask right?

Reply:
If you mean the full/partial schema reask, that hasn't been merged yet (see https://github.com/ShreyaR/guardrails/pull/248)

I'm fuzzy on exactly how the old pydantic integration worked, but from my understanding it reasked in full; the new pydantic integration compiles to XML and uses the XML reask internals (therefore partial)

Reply:
Ok, why did the output schema change then (other than removing the pydantic tag)? Just to make the test simpler?